modules/composer-ffmpeg/src/main/java/org/opencastproject/composer/impl/endpoint/ComposerRestService.java

```diff
@@ -494,14 +494,16 @@ public Response composite(@FormParam("compositeSize") String compositeSizeJson,
   @RestQuery(name = "concat", description = "Starts a video concating process from multiple videos, based on the specified encoding profile ID and the source tracks", restParameters = {
           @RestParameter(description = "The source tracks to concat as XML", isRequired = true, name = "sourceTracks", type = Type.TEXT),
           @RestParameter(description = "The encoding profile to use", isRequired = true, name = "profileId", type = Type.STRING),
-    @RestParameter(description = "The resolution dimension of the concat video as JSON", isRequired = false, name = "outputDimension", type = Type.STRING),
-    @RestParameter(description = "The  frame rate of the concat video (should be positive, e.g. 25.0). Negative values and zero will deactivate frame rate operation.",
-            isRequired = false, name = "outputFrameRate", type = Type.STRING)}, reponses = {
+          @RestParameter(description = "The resolution dimension of the concat video as JSON", isRequired = false, name = "outputDimension", type = Type.STRING),
```

Comment:
A callout, or a (very) brief example would be nice here.  Are we talking "dimension: 800x600"?  Or { height: 800, width: 600}?  A link to the docs would be good here.